### PR TITLE
ENH TagInserter handles snippet provider exceptions gracefully

### LINKED ISSUE #86: Adding GA or CrazyEgg tags causes fatal errors without the right metadata
Issue with tagmanager module, but flagging it here since it affects demo users. We shouldn't throw an exception and break every page rendering when users add a tag through the CMS UI without filling in a form field.

### PR DESCRIPTION
**Pull request by dnsl48**
Fixes https://github.com/silverstripe/bambusa-installer/issues/86
Partially addresses https://github.com/sminnee/silverstripe-tagmanager/issues/5

### CODE REVIEW

File: src/Model/Snippet.php [hunk 1]
```diff
@@ -66,12 +66,19 @@ public function getSnippetSummary()
     {
         $provider = $this->getSnippetProvider();
         if ($provider) {
-            return $provider->getSummary((array)json_decode($this->SnippetParams, true));
+            return $provider->getSummary((array) json_decode($this->SnippetParams, true));
```
Comment (sminnee) [inline src/Model/Snippet.php:69]: Does this change meet the needs of some style guide?

**Review by sminnee — APPROVED** [review]
Looks good to me, one question about a random addition of a space :p

File: src/Model/Snippet.php (hunk 1 above)
Comment (dnsl48) [reply src/Model/Snippet.php:69]: Well, that's not really a requirement, but rather how they do it in the [PSR-12 examples](https://www.php-fig.org/psr/psr-12/#61-unary-operators).
I didn't intend to include this change on its own, but it was rather a part of some other refactoring (I wanted to move this block into a separate private method), but then decided not to include that into this patch, and reverted back, but not completely - forgot to remove this space :)

**Comment by sminnee** [comment]
:+1:

**Pull request merged**